packages/e2e-tests/specs/widgets/editing-widgets.test.js

```diff
@@ -394,6 +394,111 @@ describe( 'Widgets screen', () => {
 	` );
 	} );
 
+	async function addMarquee() {
+		// There will be 2 matches here.
+		// One is the in-between inserter,
+		// and the other one is the button block appender.
+		const [ inlineInserterButton ] = await findAll( {
+			role: 'combobox',
+			name: 'Add block',
+		} );
+		await inlineInserterButton.click();
+
+		// TODO: Convert to find() API from puppeteer-testing-library.
+		const inserterSearchBox = await page.waitForSelector(
+			'aria/Search for blocks and patterns[role="searchbox"]'
+		);
+		await expect( inserterSearchBox ).toHaveFocus();
+
+		await page.keyboard.type( 'Marquee' );
+
+		const inlineQuickInserter = await find( {
+			role: 'listbox',
+			name: 'Blocks',
+		} );
+		const marqueeBlockOption = await find(
+			{
+				role: 'option',
+			},
+			{
+				root: inlineQuickInserter,
+			}
+		);
+		await marqueeBlockOption.click();
+	}
+
+	it( 'Should add and save the marquee widget', async () => {
+		await activatePlugin( 'gutenberg-test-marquee-widget' );
+		await visitAdminPage( 'widgets.php' );
+
+		await addMarquee();
+
+		await find( {
+			selector: '[data-block][data-type="core/legacy-widget"]',
+		} );
+
+		const greetingsInput = await find( {
+			selector: '#marquee-greeting',
+		} );
+		await greetingsInput.click();
+		await page.keyboard.type( 'Howdy' );
+
+		await saveWidgets();
+
+		let editedSerializedWidgetAreas = await getSerializedWidgetAreas();
+		await expect( editedSerializedWidgetAreas ).toMatchInlineSnapshot( `
+		Object {
+		  "sidebar-1": "<marquee>Hello!</marquee>",
+		}
+	` );
+
+		await page.reload();
+
+		editedSerializedWidgetAreas = await getSerializedWidgetAreas();
+		await expect( editedSerializedWidgetAreas ).toMatchInlineSnapshot( `
+		Object {
+		  "sidebar-1": "<marquee>Hello!</marquee>",
+		}
+	` );
+
+		// Add another marquee, it shouldn't be saved
```

Comment:
```suggestion
		// Add another marquee
```

Nit: this bit doesn't perform any assertions on whether it's saved. As a result, this comment confused me slightly.